packages/matter.js/src/device/Aggregator.ts

```diff
@@ -17,26 +18,19 @@ import { ClusterServer } from "../protocol/interaction/InteractionServer.js";
  * If Power source information should be provided you need to also add the needed clusters (PowerSourceConfigurationCluster
  * and PowerSourceCluster) to the device!
  */
-export class Aggregator extends ComposedDevice {
+export class Aggregator extends Endpoint {
     /**
      * Create a new Aggregator instance and optionally directly add devices to it. If this is used the devices must
      * already have the BridgedDeviceBasicInformationCluster added!
      * @param devices Array of devices to add
-     * @param endpointId Optional endpoint ID to use. If not provided will automatically be assigned
+     * @param options Optional Endpoint options
```

Comment:
Optional Endpoint options seems a bit redundant ;)

Reply:
:-) yes but name, and comment :-)

Reply:
Yeah just meant the comment, could be "Optional endpoint settings" or "Optional endpoint parameters".  Anyway pretty trivial, going to mark resolved